.github/actions/prepare-node-deps/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: deps
+description: Common deps for mobile repo
+
+runs:
+  using: composite
+  steps:
+    - name: ci/setup-node
+      uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+      with:
+        node-version: "${{ env.NODE_VERSION }}"
+        cache: "npm"
+        cache-dependency-path: package-lock.json
+
+    - name: ci/install-npm-dependencies
+      shell: bash
+      env:
+        NODE_ENV: development
+      run: |
+        echo "::group::install-npm-dependencies"
+        npm ci --ignore-scripts
+        node node_modules/\@sentry/cli/scripts/install.js
+        node node_modules/react-native-webrtc/tools/downloadWebRTC.js
```

Comment:
after updating to the latest react-native-webrtc library this step is no longer needed.